include/posix/pthread_sched.h

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
These definitions are in a file named `<sched.h>`; shouldn't them be in a file with the same name so that code that assumes this will be able to find them without change?